pyvirtualcam/camera.py

```diff
@@ -92,19 +97,54 @@ def sleep_until_next_frame(self) -> None:
                 time.sleep(t_sleep)
 
 
-class _WindowsCamera(CameraBase):
+class _NativeCamera(CameraBase):
     def __init__(self, width: int, height: int, fps: float, delay=10, print_fps=False) -> None:
         super().__init__(width, height, fps, delay, print_fps)
-        _native_windows.start(width, height, fps, delay)
+        _native.start(width, height, fps, delay)
 
     def close(self) -> None:
         super().close()
-        _native_windows.stop()
+        _native.stop()
 
     def send(self, frame: np.ndarray) -> None:
         super().send(frame)
-        _native_windows.send(frame)
+        _native.send(frame)
 
 
-if platform.system() == 'Windows':
-    Camera = _WindowsCamera
+class _PyFakeWebcamCamera(CameraBase):
+    def __init__(self, width: int, height: int, fps: float, delay=10, print_fps=False) -> None:
+        super().__init__(width, height, fps, delay, print_fps)
+        self._pyfakewebcam = None
+        index = 0
+        while index < 100:
+            video_device = f'/dev/video{index}'
+            if os.path.exists(video_device):
+                try:
+                    self._pyfakewebcam = pyfakewebcam.FakeWebcam(video_device, width, height)
+                    return
+                except OSError as err:
+                    if err.errno == 22: # Invalid argument
+                        # Usually happens if a camera exists, but it's not a v4l2loopback device
+                        pass
+                    else:
+                        assert(False)
+                except:
```

Comment:
I don't think this serves much value, esp since it will push the original error further up in the console output.